lib/elastomer/client/docs.rb

```diff
@@ -275,45 +275,9 @@ def count( query, params = nil )
       # Returns a Hash of statistics about the delete operations
       def delete_by_query( query, params = nil )
         query, params = extract_params(query) if params.nil?
+        params = { :index => @name, :type => @type }.merge params
```

Comment:
Use the `update_params` method here, as in https://github.com/github/elastomer-client/blob/command-object/lib/elastomer/client/docs.rb#L294